pyupgrade/_plugins/native_literals.py

```diff
@@ -19,15 +20,15 @@
 SIX_NATIVE_STR = frozenset(('ensure_str', 'ensure_text', 'text_type'))
 
 
-def _fix_native_str(i: int, tokens: list[Token]) -> None:
+def _fix_native_literal(i: int, tokens: list[Token], *, empty: str) -> None:
```

Comment:
sorry!  one more thing, let's just call this `_fix_literal` since it has nothing to do with `native` now